src/viur/datastore/utils.py

```diff
@@ -140,7 +140,7 @@ def startDataAccessLog() -> Set[Union[Key, str]]:
 
 def endDataAccessLog(outerAccessLog: Optional[Set[Union[Key, str]]] = None) -> Optional[Set[Union[Key, str]]]:
 	"""
-		Retrieves the set of entries accessed so far. To start the log, call :func:`viur.datastore.startAccessDataLog`.
+		Retrieves the set of entries accessed so far. To start the , call :func:`viur.datastore.startAccessDataLog`.
```

Comment:
"log" was removed, but the sentence doesn't make sense then.
```suggestion
		Retrieves the set of entries accessed so far. To start the log, call :func:`viur.datastore.startAccessDataLog`.
```